managers/s3.py

```diff
@@ -37,11 +42,79 @@ def createS3Bucket(self, bucketName, bucketPermissions):
         raise S3Error('Unable to create bucket in s3')
 
 
-    def createManifestInS3(self, manifestPath, manifestJSON, s3_bucket: str):
+    def store_pdf_manifest(self, record, bucket_name, requires_login: bool):
+        for link in record.has_part:
+            item_no, uri, source, media_type, _ = link.split('|')
+
+            if media_type == 'application/pdf':
+                record_id = record.identifiers[0].split('|')[0]
+
+                if requires_login:
+                    manifest_path = f'manifests/publisher_backlist/{source}/{record_id}.json'
```

Comment:
We should generalize this logic for other sources. We could consider passing in an optional prefix.